cmd/utils/flags.go

```diff
@@ -138,6 +138,11 @@ var (
 		Name:  "olympic",
 		Usage: "Use olympic style protocol",
 	}
+	EthVersionFlag = cli.IntFlag{
+		Name:  "eth",
+		Value: 61,
```

Comment:
So 62 is not advertised? Is it planned to be advertised at some time after 62 is deployed?